styles/typography.css

```
@@ -14,5 +14,5 @@
 
 :root {
   --font-family-serif: "Source Serif 4", serif;
-  --bs-font-sans-serif: "Source Sans 3";
+  --bs-font-sans-serif: "Source Sans 3", system-ui, -apple-system, "Segoe UI", roboto, "Helvetica Neue", "Noto Sans", "Liberation Sans", arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```

```suggestion
  --bs-font-sans-serif: "Source Sans 3", system-ui;
```

system-ui is broadly supported. I can't think of a case where we'd need more than this.

---

I believe I copied the upstream font stack and pre-pended our brand font. That bootstrap hasn't dropped the fallbacks seems like an indicator that it's maybe too soon?